### ISSUE TITLE
Make Diversity program area field uniform

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#4118]
### Overview
We want to change the wording for Write For All's program area to `Diversity, Equity & Inclusion` so that it is consistent with the wording used in New Schools Today's program area and we maintain a professional looking site.

### Action Items
- [x] In `_projects/writeforall.md`, change 
  ```
  - Diversity / Equity and Inclusion
  ```
  to
  ```
  - Diversity, Equity & Inclusion
  ```
- [x] Ensure that the program area is displayed properly on the Write for All card on the [Projects page](https://www.hackforla.org/projects/)

#### Merge Team, Technical Lead, or Product Manager
- [x] Once the pull request associated with this issue is merged, go to issue #4120 and check if all the issues listed under the `Dependency` section are closed. If they are all closed, move that issue (#4120) into the `New Issue Approval` column, remove the `Dependency` label on it, and add a `ready for dev lead` label to it.

### Resources/Instructions
- [_projects/writeforall.md](https://github.com/hackforla/website/blob/gh-pages/_projects/writeforall.md?plain=1)
- [Projects page](https://www.hackforla.org/projects/)

**Comment by Alsuang** [comment]
Availability: Monday 9-12pm
ETA: Monday EOD

**Comment by ExperimentsInHonesty** [comment]
@hackforla/website-merge After reviewing this Merge, I realized that the fix made the two DEI programs uniform, but the program area drop down is not working on the projects page https://www.hackforla.org/projects.

If you go to the projects page and try to use any of the Program Areas, they all work 
https://www.hackforla.org/projects/?programs=Citizen+Engagement
https://www.hackforla.org/projects/?programs=Civic+Tech+Infrastructure
https://www.hackforla.org/projects/?programs=Environment
https://www.hackforla.org/projects/?programs=Justice
https://www.hackforla.org/projects/?programs=Social+Safety+Net
https://www.hackforla.org/projects/?programs=Vote+/+Representation
https://www.hackforla.org/projects/?programs=Workforce+Development

Except for Diversity, Equity & Inclusion because it has an ampersand in it. 
https://www.hackforla.org/projects/?programs=Diversity,+Equity+&+Inclusion

Let's make a new small issue to change both pages to Diversity / Equity / Inclusion and see if that makes the drop down work.  Its similar to Vote / Representation so I anticipate it will.

**Comment by JessicaLucindaCheng** [comment]
@ExperimentsInHonesty 
- Issue #4120 is to fix the problem with the dropdown and someone is currently assigned to that issue. Since you want to change the Program Area to be `Diversity / Equity / Inclusion`, what do you want to do with #4120?

**Comment by t-will-gillis** [comment]
@ExperimentsInHonesty @JessicaLucindaCheng  
I am working on #4120 - Yes "Diversity / Equity / Inclusion" will work - I just checked. 

(The previous "Diversity, Equity & Inclusion" had problems not only because of the "&" but also because of the ",". Even removing the ampersand, with the comma the dropdown was splitting into two unique program areas of "Diversity" and "Equity and Inclusion"  . )

**Comment by ExperimentsInHonesty** [comment]
@JessicaLucindaCheng @t-will-gillis I will close this issue.  I have left a note on https://github.com/hackforla/website/issues/4120#issuecomment-1479808595 to go ahead and change the Program Area to be Diversity / Equity / Inclusion
